certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -1032,8 +1032,8 @@ def more_info(self):
     def auth_hint(self, failed_achalls): # pragma: no cover
         return (
             "The Certificate Authority failed to verify the temporary nginx configuration changes "
-            "made by the --nginx plugin. Ensure the above domains point to this nginx server and "
-            "that it is accessible from the internet, or try increasing --nginx-sleep-seconds."
+            "made by Certbot. Ensure the listed domains point to this nginx server and that it is "
+            "accessible from the internet."
```

Comment:
Why no more mention of `--nginx-sleep-seconds`?

Reply:
I'm not strongly against including it, but my reflection on it is that it is too unlikely a culprit, to warrant inclusion here. We've encountered only a tiny number of users for which it was relevant. I think you put it to me as the 80/20 principle before (but I'd guess it's more like 98/2 here).

The nginx authenticator seems pretty damned reliable overall and hopefully #8663 fixes the 2 most common ways I see it fail. I don't know whether there's any troubleshooting advice we can give besides what we have, other than describing it in a more in-depth way.

We can always re-add it (and anything else that comes up!), if it does turn out to be useful.

Reply:
Great argument, I'm satisfied.